charts/open5gs-webui/templates/deployment.yaml

```diff
@@ -0,0 +1,112 @@
+      {{- if .Values.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAffinityPreset "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.podAntiAffinityPreset "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.nodeAffinityPreset.type "key" .Values.nodeAffinityPreset.key "values" .Values.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      serviceAccountName: {{ include "open5gs.webui.serviceAccountName" . }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName }}
+      {{- end }}
+      {{- if .Values.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.topologySpreadConstraints "context" .) | nindent 8 }}
+      {{- end }}
+      priorityClassName: {{ .Values.priorityClassName | quote }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.hostAliases }}
+      # yamllint disable rule:indentation
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.hostAliases "context" $) | nindent 8 }}
+      # yamllint enable rule:indentation
+      {{- end }}
+      initContainers:
+      - name: init
+        image: bitnami/mongodb:4.4.1-debian-10-r39
+        env:
+        - name: DB_URI
+          {{- if .Values.dbURI }}
+          value: {{ .Values.dbURI }}
+          {{- else  }}
+          value: {{ printf "mongodb://%s/open5gs" (include "common.names.dependency.fullname" (dict "chartName" "mongodb" "chartValues"  .Values.mongodb "context" $)) }}
+          {{- end }}
+        command: 
+          - /bin/bash
+          - /add_admin.sh
+        volumeMounts:
+        - name: populate
+          mountPath: /add_admin.sh
+          subPath: add_admin.sh
+      containers:
+      - name: webui
+        image: {{ template "open5gs.webui.image" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+        env:
+        - name: DB_URI
+          {{- if .Values.dbURI }}
+          value: {{ .Values.dbURI }}
+          {{- else  }}
+          value: {{ printf "mongodb://%s/open5gs" (include "common.names.dependency.fullname" (dict "chartName" "mongodb" "chartValues"  .Values.mongodb "context" $)) }}
+          {{- end }}
+        ports:
+        - name: http
+          containerPort: {{ .Values.containerPorts.http }}
+          protocol: TCP
+        livenessProbe:
```

Comment:
Fix this to use livenessProbe from values.yaml and readinessProbe from values.yaml.

Use [bitnami drupal helm chart](https://github.com/bitnami/charts/blob/7c940b6a734af8363df4de66d767e274436fc67c/bitnami/drupal/templates/deployment.yaml#L244) as example